ublox/src/error.rs

```diff
@@ -45,7 +45,7 @@ impl fmt::Display for ParserError {
         match self {
             ParserError::InvalidChecksum { expect, got } => write!(
                 f,
-                "Not valid packet's checksum, expect {:x}, got {:x}",
+                "Not valid packet's checksum, expect 0x{:02x}, got 0x{:02x}",
```

Comment:
emphasize this is hexa encoding/printing

Reply:
Do you prefer that way of formatting it? Otherwise `{:#02x}` achieves the same as `0x{:02x}`

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=28034d7258915efabaa04f7964cb117e

Reply:
The 0x prefix is important